app/src/modules/settings/routes/data-model/field-detail/field-detail-advanced/field-detail-advanced-validation.vue

```diff
@@ -35,6 +18,44 @@ const { t } = useI18n();
 
 const validation = syncFieldDetailStoreProperty('field.meta.validation');
 const validationMessage = syncFieldDetailStoreProperty('field.meta.validation_message');
+
+const validationSync = computed({
+	get() {
+		return {
+			...(validation.value && { validation: validation.value }),
+			...(validationMessage.value && { validationMessage: validationMessage.value }),
+		};
+	},
+	set(value) {
+		validation.value = value.validation;
+		validationMessage.value = value.validationMessage;
```

Comment:
```suggestion
		validation.value = value.validation ?? null;
		validationMessage.value = value.validationMessage ?? null;
```

Tiny tweak since in the `get()`, we do not return `validation` or `validationMessage` if they're falsy, which includes `undefined`. We can then get `undefined` value for them instead of the usual `null`:

![](https://github.com/directus/directus/assets/42867097/0c069a1e-df21-45bd-bf1a-d7b89a933285)

so adding `?? null` helps make sure they are "reset" to null 👍

Reply:
Good catch 🎣